Explorer/Assets/DCL/LOD/Systems/LODUtils.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Generic;
+using CommunicationData.URLHelpers;
+using DCL.AvatarRendering.AvatarShape.ComputeShader;
+using DCL.AvatarRendering.AvatarShape.Rendering.TextureArray;
+using DCL.Diagnostics;
+using DCL.Optimization.Pools;
+using ECS.SceneLifeCycle.SceneDefinition;
+using SceneRunner.Scene;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+
+namespace DCL.LOD
+{
+    public static class LODUtils
+    {
+        public static readonly URLDomain LOD_WEB_URL = URLDomain.FromString("https://ab-cdn-decentraland-org-contentbucket-4e8caab.s3.amazonaws.com/LOD/");
```

Comment:
isnt this the same as `https://ab-cdn.decentraland.org/LOD/` ?

Reply:
Why the single bucket is used?